Modules/Devices/src/Devices.Infrastructure/PushNotifications/DirectPush/PnsConnectorFactory.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Backbone.Modules.Devices.Domain.Aggregates.PushNotifications;
+
+namespace Backbone.Modules.Devices.Infrastructure.PushNotifications.DirectPush;
+public abstract class PnsConnectorFactory
+{
+    public static IPnsConnector CreateFor(PushNotificationPlatform platform)
```

Comment:
Why is this method static? How would it call the instance methods (CreateForFirebaseCloudMessaging and CreateForApplePushNotificationService) of the `PnsConnectorFactoryImpl` class?